examples/02_control_flow/01_When.md

```diff
@@ -1,6 +1,8 @@
 # When
 
-Kotlin does not have `switch` statement, instead of it it has more smart and clear `when` construction. Let's take a look
+Instead of widely used `switch` statement, Kotlin provides the more flexible and clear `when` construction. It can be used either as a statement or as an expression.
```

Comment:
Instead of the widely used `switch` statement, Kotlin provides the more flexible and clear `when` construction. It can be used either as a statement or as an expression.